examples/jquery/__tests__/displayUser-test.js

```diff
@@ -1,5 +1,7 @@
 // Copyright 2004-present Facebook. All Rights Reserved.
 
+/* global document */
```

Comment:
don't really need it if it is only used in two tests in the entirety of Jest. Jest's other tests don't use jsdom, so document isn't available.